.github/workflows/cd-explore-stage.yml

```diff
@@ -0,0 +1,49 @@
+name: CI Component Explorer
+
+on:
+  push:
+    branches:
+    - master
+
+jobs:
+  build_linux:
+    runs-on: ubuntu-latest
+
+    env:
+      AZURE_WEBAPP_NAME: fast-explore
+      AZURE_WEBAPP_PACKAGE_PATH: ./packages/fast-component-explorer/www
+      AZURE_WEBAPP_SLOT_NAME: stage
+
+    steps:
+    - name: Azure Login
+      uses: azure/login@v1
+      with:
+        creds: ${{ secrets.AZURE_CREDENTIALS_FAST_DESIGN }}
+
+    - name: Checkout Branch
+      uses: actions/checkout@master
+
+    - name: Install Lerna
+      run: sudo yarn global add lerna
+
+    - name: Install package dependencies
+      run: yarn install --frozen-lockfile
+
+    - name: Prepare Yarn workspaces
```

Comment:
I *believe* yarn will actually prepare on install so this `lerna run prepare` step might not be necessary

Reply:
Yeah I think this may be accurate, especially if done in the root.